kubemarine/procedures/check_paas.py

```diff
@@ -385,12 +385,20 @@ def thirdparties_hashes(cluster):
             # SHA is correct, now check if it is an archive and if it does, then also check SHA for archive content
             if 'unpack' in config:
                 unpack_dir = config['unpack']
-                # TODO support zip
-                res = group.sudo('tar tf %s | grep -vw "./" | while read file_name; do '  # for each file in archive
+                arr = path.split(".") 
```

Comment:
If some thirdparty is in gz archive (f.e., `filename.gz`) it will not be able to get unpacked by `unzip`.

Reply:
Changed the logic for if condition so that it could check the extension of the file.